### ISSUE TITLE
Add ban notice to top of appeals if active ban

### ISSUE THREAD

**Issue by dqwiki** [UTRS2/utrs#238]
It would be nice to be able to look at an appeal and know if there is an active ban on the appealfor target. Don't worry about the hiddenip.

**Comment by DeirgeDel** [comment]
I have been impacted by this not actioned.  I was about to raise the following as a new issue but I'm going to add it here instead.

This use case here is for people who have to submit UTRS cases via the https://utrs-beta.wmflabs.org/public/appeal/account due to being banned from the talk page.

1) Users who are banned and barred from the talk page should not prevented from entering information into the "Why should you be unblocked?" text box or alternative be permitted to check to see if they are prevented from raising a UTRS in the first place.
* My understanding is WMF has a duty to minimize mental stress to its contributors, even to those it has elected to block.
* There is mental stress preparing a UTRS appeal
* There is mental stress it may be rejected.
* To see such an appeal filed in  /dev/null or wherever is an extreme mental stress event that risks triggering an unnecessary psychotic episode on the appellant.

I confess I am not entirely sure how a appellant knows or can check they are under a UTRS ban period.  In my case my best best was it was 6 months after what I understood to be my last appeal which should have expired 12-02-2023 according to my calculations.

Your IP address or username has been banned from using UTRS. The banning administrator specified the following reason: "needs to wait six months". The ban expires on 2023-03-05 00:00:00. If you contact UTRS admininstrators about this ban, please mention the following ban ID: #663.

I feel particularly aggrieved by this have been prevented from raising a UTRS appeal due to a barf at my ideal selected timeslot, totally accepting the barf issue there and how it was dealt with, fairly inconvenienced at then shoehorning my appeal in at a less than ideal timeslot but then very annoyed at it being rejected.

PS: By the way "admininstrators" is spelt wrong in the above error message.  As that's the sort of mistake I often make I won;t complain about it.
